ex/loqui/lib/loqui/ranch_protocol.ex

```diff
@@ -313,22 +299,26 @@ defmodule Loqui.CowboyProtocol do
   end
 
   @spec handler_push(state, binary) :: :ok
+  defp handler_push(%{handler: handler, codec: codec, handler_supports_push: true}=state, request) do
```

Comment:
Was this just not used in the past? 

If the handler doesn't support push and the client tries to do a push, what will the client receive?

Reply:
The prior implementation (which is still there for handlers that don't have this function) is to call `loqui_request`.

Reply:
I see, I misread 👍

Reply:
It wasn't shown in the diff, it's easy to not see.